templates/definition/meter/sungrow-sgx0rt.yaml

```diff
@@ -0,0 +1,36 @@
+template: sungrow-sgx0rt
```

Comment:
sungrow-inverter?

Reply:
I'm not totally sure if these registers are good for other inverter models. I have the SG8.0RT and the number describes the power potential.

I don't have strong opinions about the name so if you have a naming schema you like better that's fine with me.

Maybe then a bit more description about the supported model here and in the hybrid case? Hybrid inverters start with SH

Reply:
Schau mal bitte obs so passt. Ich hab das Wording und die Dateinamen überall gleich gezogen.